### PR TITLE
Add lakes alias to Figure.coast()

### PR DESCRIPTION
**Pull request by willschlitzer**
Adding the `lakes` alias for the **-C** argument to `Figure.coast()`.

### CODE REVIEW

File: pygmt/base_plotting.py
Comment (seisman) [inline pygmt/base_plotting.py:117]:
```suggestion
            default is the fill chosen for wet areas set by the ``water`` argument.
```

File: pygmt/base_plotting.py
Comment (seisman) [inline pygmt/base_plotting.py:119]:
```suggestion
            Optionally, specify separate fills by appending **+l** for lakes or
            **+r** for river-lakes, and passing multiple strings in a list.
```

**Comment by seisman** [comment]
Also see some discussions in https://github.com/GenericMappingTools/pygmt/pull/474#discussion_r434281630.

Ping @GenericMappingTools/python for comments on the argument name.

**Comment by willschlitzer** [comment]
@seisman Looking at that PR, I'm not sure what the issue with using a long-name alias for `C` is.  It seems like the main concern is backwards compatibility, but wouldn't scripts still be able to use `C` instead of the alias?

**Comment by seisman** [comment]
> It seems like the main concern is backwards compatibility, but wouldn't scripts still be able to use `C` instead of the alias?

Yes, `C` always works.

We need to be very careful about choosing these long names. Sometimes we may find that the long names are inappropriate and have to change them to other names, which may break backward compatibility. For example, @weiji14 and I were thinking about changing `sizes` to `size` in `plot()` (https://github.com/GenericMappingTools/pygmt/pull/666#issuecomment-716084019).

**Comment by seisman** [comment]
Just checked the documentation of `coast()`, I think `lakes` is a good name, considering that we already use "rivers", "shorelines" and "borders".

**Review by seisman — APPROVED** [review]
OK to merge after you fix this style issue:
```
pygmt/base_plotting.py:117:80: W505 doc line too long (83 > 79 characters)
```

**Comment by seisman** [comment]
@willschlitzer When merging a PR, it would be better to simplify the commit description. For example, this is how this PR looks like after merging into the master branch. 

image

A better one is:

image

**Comment by willschlitzer** [comment]
@seisman Sounds good; my apologies for just leaving the defaults in.

**Pull request merged**
